pom.xml

```diff
@@ -157,7 +157,8 @@ under the License.
 		<beam.version>2.43.0</beam.version>
 		<protoc.version>3.21.7</protoc.version>
 		<okhttp.version>3.14.9</okhttp.version>
-		<testcontainers.version>1.17.2</testcontainers.version>
+		<!-- keep jna version in sync -->
```

Comment:
```suggestion
		<!-- keep net.java.dev.jna:jna version in sync -->
```
nit: shall we be a bit more specific here? I'm just afraid that the term "jna" is too generic to find the right location in a simple grep

Reply:
I think it makes sense, thanks for the suggestion!